src/zepben/evolve/services/network/tracing/feeder/set_direction.py

```diff
@@ -77,40 +81,35 @@ def next_direction_func():
             return next_direction
         return FeederDirection.NONE
 
-    async def _create_traversal(self, state_operators: Type[NetworkStateOperators]) -> NetworkTrace[FeederDirection]:
+    def _create_traversal(self, state_operators: Type[NetworkStateOperators]) -> NetworkTrace[FeederDirection]:
```

Comment:
are there consequences in making this not async anymore? Or were we never using it async in the first place?